replay-protected-sighash.md

```diff
@@ -89,7 +95,7 @@ The sighash type is altered to include a 24-bit *fork id* in its most significan
   ss << ((GetForkID() << 8) | nHashType);
 ````
 
-This ensure that the proposed digest algorithm will generate different results on forks using different *fork ids*.
+This ensure that the proposed digest algorithm we use when SIGHASH_FORKID is set will generate different results on forks using different *fork ids*.
```

Comment:
Repeating when SIGHASH_FORKID all over the place do not improve the spec. It's already mentioned that this algorithm is used when SIGHASH_FORKID is set, repeating it all the place in the description of the algorithm do not improve anything. It just dilute the message, also makes updating the spec harder than it needs to.